tests/php/Integration/Compiler/Reader/ReaderTest.php

```diff
@@ -758,10 +759,10 @@ public function testShortFnRestArgumentMultipleTimes(): void
     /**
      * @return TypeInterface|string|float|int|bool|null
      */
-    private function read(string $string, bool $removeLoc = false)
+    private function read(string $string, bool $withLocation = false)
```

Comment:
Shouldn't the the default argument of `$withLocation` also change to `true`

Reply:
Yes, why not. That thing + changing the line below where this is used. I will do that.
```php
$tokenStream = $this->compilerFacade->lexString($string, Lexer::DEFAULT_SOURCE, $withLocation); # <------
```